server/src/main/java/com/google/coffeehouse/storagehandler/StorageHandler.java

```diff
@@ -67,9 +67,11 @@ public static Person getPerson(DatabaseClient dbClient, String userId) {
               Key.of(userId),
               Arrays.asList("email", "nickname", "pronouns"));
     if (row != null) {
-      Person.Builder personBuilder = Person.newBuilder(row.getString(/*emailIndex=*/0),
-                                                       row.getString(/*nicknameIndex=*/1));
-      // TODO: implement setting the userId field @JosephBushagour
+      Person.Builder personBuilder = Person.newBuilder()
+                                           .setEmail(row.getString(/*emailIndex=*/0))
```

Comment:
There should be a spaces in this line: `/* emailIndex= */ 0`. Please update here and the line below.

Reply:
Good catch! All parameters comments in this file were updated to fit these guidelines.

Reply:
This was my error -- sorry about that, and thank you for fixing that Joey!